test/c_api/disjoint_pool.c

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2023 Intel Corporation
+// Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <stdlib.h>
+
+#include "provider_null.h"
+#include "test_helpers.h"
+#include "pool/pool_disjoint.h"
+
+void test_disjoint_pool_default_params() {
+    umf_memory_provider_handle_t provider = nullProviderCreate();
+    enum umf_result_t retp;
+    umf_memory_pool_handle_t pool = NULL;
+    struct umf_disjoint_pool_params params = umf_disjoint_pool_params_default();
```

Comment:
could you declare this func with using cammel-case style?

Reply:
Done